crates/env/src/error.rs

```diff
@@ -12,41 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use derive_more::From;
+/// Errors that can be encountered upon interaction with the off-chain environment.
+#[cfg(feature = "ink-experimental-engine")]
+pub type Error = ink_engine::Error;
```

Comment:
Note that an off-chain engine is ideally agnostic to the layer on top. So the off-chain environment should ideally have no mentions or knowledge about ink!. It could even live in its own repository and ink! just "randomly" depends on it as with SEAL.